config/locales/hyrax.zh.yml

```diff
@@ -317,11 +317,11 @@ zh:
         resource_type: 您可以选择适用于所有文件的多种类型
         title: 文件名将作为默认标题。请提供更有意义的标题，文件名将会在系统内保留。
     batch_uploads:
-      disabled: 功能由管理员禁用
+      disabled: 功能被管理员禁用
       files:
-        button_label: 添加新作品
         instructions: 上传的每份文件都会分别按新作品生成。
-        upload_type_instructions: 如果建立一件作品包括所有文件，请去
+        upload_type_instructions: 创建一件包括所有文件的作品，请去
+        button_label: 添加新作品
```

Comment:
Please move `button_label` back to its prior location. (This file is ordered.) Thanks!